mozetl/symbolication/top_signatures_correlations.py

```diff
@@ -0,0 +1,158 @@
+# Migrated from Databricks to run on dataproc
+# pip install:
+# boto3==1.16.20
+# scipy==1.5.4
+
+import argparse
+import hashlib
+import os
+import sys
+from collections import defaultdict
+from datetime import datetime
+
+from pyspark import SparkContext
+from pyspark.sql import SparkSession
+
+sys.path += [os.path.abspath("."), os.path.abspath("crashcorrelations")]
+os.system("git clone https://github.com/marco-c/crashcorrelations.git")
```

Comment:
The repos these jobs use aren't setup to be installed via pip so they have these weird lines.